### PR TITLE
Add section for Godcr download

### LINKED ISSUE #27: add a section to download godcr on the downloads page

### PR DESCRIPTION
**Pull request by devchoplife**
Closes #27 
This PR adds a section on the downloads page to download godcr

image

### CODE REVIEW

File: layouts/downloads/list.html [hunk 1]
```diff
@@ -15,6 +15,11 @@ <h2 >Decred Wallet For Mobile</h2>
         <a href="https://play.google.com/store/apps/details?id=com.decred.dcrandroid.mainnet" target="_blank"><img src="{{ "../images/google-store-icon.png" | absURL }}" alt="AndroidApp"></a>
         <a href="https://apps.apple.com/us/app/decred-wallet/id1462247643" target="_blank"><img src="{{ "../images/apple-store-icon.png" | absURL }}" alt="iOSAPP"></a>
     </div>
+
+    <div class="downloads-container">
+        <h2 >Decred Wallet For Computers</h2>
```
Comment (dreacot) [inline layouts/downloads/list.html]: GoDCR - A cross-platform desktop [SPV](https://docs.decred.org/wallets/spv/) wallet for [decred](https://decred.org/) built with [gio](https://gioui.org/)

File: layouts/downloads/list.html [hunk 2]
```diff
@@ -15,6 +15,11 @@ <h2 >Decred Wallet For Mobile</h2>
...
+        <a href="https://github.com/planetdecred/godcr/releases" target="_blank">Download GoDCR binaries</a>
```
Comment (dreacot) [inline layouts/downloads/list.html]: Screenshot from 2022-05-18 22-48-29

Let's have download buttons for individual platforms, so they can download directly

something like in the picture above

**Review by dreacot — CHANGES_REQUESTED** [review]

**Comment by raedah** [comment]
Make sure the fonts sizes are reasonable with good proportions.

**Comment by raedah** [comment]
Also note vertical spacing between sections.

**Comment by raedah** [comment]
Mobile buttons can be put side by side since we are adding more to the page.

**Comment by dreacot** [comment]
> Mobile buttons can be put side by side since we are adding more to the page.

@devchoplife this hasn't been attended to, i believe he is referring to the mobile buttons

File: layouts/downloads/list.html [hunk 3]
```diff
@@ -15,6 +15,18 @@ <h2 >Decred Wallet For Mobile</h2>
         <a href="https://play.google.com/store/apps/details?id=com.decred.dcrandroid.mainnet" target="_blank"><img src="{{ "../images/google-store-icon.png" | absURL }}" alt="AndroidApp"></a>
         <a href="https://apps.apple.com/us/app/decred-wallet/id1462247643" target="_blank"><img src="{{ "../images/apple-store-icon.png" | absURL }}" alt="iOSAPP"></a>
     </div>
+
+    <div class="downloads-container">
+        <h2 >GoDCR - Desktop Wallet For Decred</h2>
+        <img style = "height: 100px;width: 100px;" src = "{{ "../images/godcr.png" | absURL }}" alt="Decred Logo" />
+        <div>
+            <a class = "add-margin" href="https://github.com/planetdecred/godcr/releases/download/v1.7.0/GoDCR-v1.7.0.dmg" target="_blank">Mac OS ↓ <span style="margin-left: 5px;">|</span></a>
```
Comment (dreacot) [inline layouts/downloads/list.html:25]: remove the underline when the links are hovered

**Review by dreacot — CHANGES_REQUESTED** [review]

**Review by dreacot — APPROVED** [review]

File: layouts/downloads/list.html [hunk 4]
```diff
@@ -12,8 +12,22 @@
...
-        <a href="https://play.google.com/store/apps/details?id=com.decred.dcrandroid.mainnet" target="_blank"><img src="{{ "../images/google-store-icon.png" | absURL }}" alt="AndroidApp"></a>
-        <a href="https://apps.apple.com/us/app/decred-wallet/id1462247643" target="_blank"><img src="{{ "../images/apple-store-icon.png" | absURL }}" alt="iOSAPP"></a>
+        <div>
+          <a href="https://play.google.com/store/apps/details?id=com.decred.dcrandroid.mainnet" target="_blank"><img src="{{ "../images/google-store-icon.png" | absURL }}" alt="AndroidApp"></a>
+          <a href="https://apps.apple.com/us/app/decred-wallet/id1462247643" target="_blank"><img src="{{ "../images/apple-store-icon.png" | absURL }}" alt="iOSAPP"></a>
+        </div>
+    </div>
+
+    <div class="downloads-container">
+        <h2 >GoDCR - Desktop Wallet For Decred</h2>
```
Comment (dreacot) [inline layouts/downloads/list.html]:
```suggestion
        <h2 >GoDCR - Decred Wallet for Desktop</h2>
```

**Review by dreacot — CHANGES_REQUESTED** [review]

**Review by dreacot — APPROVED** [review]

**Pull request merged**